src/OpenTelemetry.Contrib.Instrumentation.EntityFrameworkCore/README.md

```diff
@@ -1,4 +1,4 @@
-# EntityFrameworkCore Instrumentation for OpenTelemetry.Contrib .NET
+# EntityFrameworkCore Instrumentation for OpenTelemetry .NET
```

Comment:
unrelated to this PR. OK with me, I hope there will be no conflicts

Reply:
TBH, it was a while ago and I don't even remember why I did this 🤣 Probably because it was inconsistent in the files already in the repo.